CHANGELOG.md

```diff
@@ -7,12 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+
+- stylize prompt to create new project or tag (#310).
```

Comment:
```suggestion
- Stylize prompt to create new project or tag (#310).
```

Reply:
:grin:

Reply:
For some reason, I though for the longest time it was the `i` that had changed, but I couldn't figure out how it was different! Then I found the `s`...

Reply:
Sorry for not being explicit!